src/test/java/catan/controllers/ctf/GameUserValidator.java

```diff
@@ -47,9 +47,9 @@ public Scenario doesntHaveAvailableAction(String action) {
         return scenario;
     }
 
-    public Scenario resourcesChanged(int brick, int wood, int sheep, int wheat, int stone) {
+    public Scenario resourcesQuantityChanged(int brick, int wood, int sheep, int wheat, int stone) {
```

Comment:
name of method doesn't say what specifically we should check here. it is not clear which parameters I should pass: new quantity of resources, or maybe the difference between old and new resources quantity